docs/advanced/api_examples.yml

```diff
@@ -355,7 +357,7 @@
     "move container"
 
   doc: |
-    Move an existing named container (and any attached datasets) to a new host.
+    Move an existing named container (and any attached datasets) to a new node.
 
   request: |
     POST /v1/configuration/containers/webserver HTTP/1.1
```

Comment:
http://build.clusterhq.com/results/docs/UUID-in-API-FLOC-1733/build-6253/advanced/api.html#post--v1-configuration-containers-%28name%29